docs/TUTORIAL.md

```diff
@@ -608,19 +630,34 @@ delegate_hashed_bins(list_of_targets, keys_of_hashed_bins, number_of_bins)
 We next provide a complete example of retrieving target paths to add to hashed
 bins, performing the hashed bin delegations, signing them, and delegating paths
 to some role.
+
 ```Python
+# Continuing from the previous section . . .
+
+# Remove 'myproject/file4.txt' from unclaimed role and instead further delegate
+# all targets in myproject/ to hashed bins.
+>>> repository.targets('unclaimed').remove_target("myproject/file4.txt")
+
 # Get a list of target paths for the hashed bins.
->>> targets = \
-  repository.get_filepaths_in_directory('repository/targets/myproject', recursive_walk=True)
->>> repository.targets('unclaimed').delegate_hashed_bins(targets, [public_unclaimed_key], 32)
+>>> targets = repository.get_filepaths_in_directory(
+...     'repository/targets/myproject', recursive_walk=True)
+
+>>> repository.targets('unclaimed').delegate_hashed_bins(
+...     targets, [public_unclaimed_key], 32)
 
 # delegated_hashed_bins() only assigns the public key(s) of the hashed bins, so
 # the private keys may be manually loaded as follows:
 >>> for delegation in repository.targets('unclaimed').delegations:
 ...   delegation.load_signing_key(private_unclaimed_key)
 
-# Delegated roles can be restricted to particular paths with add_restricted_paths().
->>> repository.targets('unclaimed').add_restricted_paths('repository/targets/myproject/*', 'django')
+
+>>> repository.dirty_roles()
+Dirty roles: ['00-07', '08-0f', '10-17', '18-1f', '20-27', '28-2f', '30-37', '38-3f', '40-47', '48-4f', '50-57', '58-5f', '60-67', '68-6f', '70-77', '78-7f', '80-87', '88-8f', '90-7', '98-9f', 'a0-a7', 'a8-af', 'b0-b7', 'b8-bf', 'c0-c7', 'c8-cf', 'd0-d7', 'd8-df', 'e0-e7', 'e8-ef', 'f0-f7', 'f8-ff', 'unclaimed']
+
```

Comment:
I get the same roles but sorted differently, which I'd expect knowing that various Python data structures have non-deterministic ordering. Does it make sense to make these deterministic (i.e. `sort()` this list)?

Reply:
Patch to sort the list before printing in `dirty_roles()` submitted in #961

Reply:
#961 was merged 🎉

Reply:
This is odd. I had already added this change in ac01033 specifically for that reason, albeit one level deeper.  Are you sure you got a different sorting?

Reply:
Oh no, I was running the steps from the `develop` branch, not a branch with the changes from this PR 🤦‍♂

We should revert/undo ff21b36. Apologies @lukpueh!

Reply:
No worries at all! :) What do you think about cherry-picking 282c9aa98b52d654f9610726a272c9c8bf34d2d1 into this PR and reverting #961?

Reply:
Sounds good to me